app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  def self.update_comments_counter(post)
+    post.comments_counter = Comment.where(post:).count
+  end
```

Comment:
- [ ] The same here, Great job creating a method that updates the comments counter for a post. 👏🏻 👏🏻  However It would be nice to use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) -* `after_save` to call the `#update_comments_counter` method every time a new like has been saved to the database, then make the `update_comments_counter` method to be a `private` method 👍🏻 💯 
- 🤔  Your code might look like this 👇🏻 :

```ruby
# app/models/comment.rb

     after_save : update_comments_counter
   
     private

    def update_comments_counter
       post.increment!(:likes_counter)
    end
```

Reply:
I didn't know about this Rails feature. I implemented it right away, and the program behaves more logically now. Thanks for the explanation!!